lib/database/migrations/20231211080706-create-data-passes.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(async (transaction) => {
+        await queryInterface.createTable('data_passes', {
+            id: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                primaryKey: true,
+                autoIncrement: true,
+            },
+            name: {
+                type: Sequelize.STRING,
+                allowNull: false,
+            },
+            description: {
+                type: Sequelize.TEXT,
+            },
+            output_size: {
+                type: Sequelize.BIGINT,
+            },
+            reconstructed_events: {
```

Comment:
For this I would put a `..._count` if it's a counter

Reply:
done